site/docs/getting-started/tutorials/continuous-materialized-view.md

```diff
@@ -37,84 +40,194 @@ First, you'll scope the raw data down to a small fact table with a derivation.
 
 You'll then materialize both the raw and transformed datasets to your Postgres instance and compare performance.
 
-## Add a derivation to transform data.
+## Check out the source data
 
-Derivations are currently available in the flowctl CLI. If you'd prefer to only work in the web app today,
-you can skip to the next section. Estuary provides a pre-computed copy of the derivation that you can use to compare performance.
+1. Got the the [collections page](https://dashboard.estuary.dev/collections) of the Flow web app.
 
-1. On the [CLI-API tab](https://dashboard.estuary.dev/admin/api) in the Flow web app, copy the access token.
+2. Search for `demo/wikipedia/recentchange` and click on its name.
 
-2. Authorize Flow in your local development environment:
+3. On the **Collection Details** page, click the **Spec** tab.
 
-   ```console
-   flowctl auth token --token <your-token-here>
+   The collection is keyed on its metadata, so every new change event is seen as unique. Its schema has many fields.
```

Comment:
_Technically_, the key of this collection is not really correct, in that it does not uniquely identify each change. The incorrectness doesn't matter in this case, because we don't reduce documents before feeding them into derivations. But in either case, I think it'd be better to just say that this collection is _quite_ large due to handling so many wikipedia edits.

It might also be a good idea to offer users the option of using the `-sampled` version of this collection, just so it goes faster and costs less. AFAIK they could just add `-sampled` to the end of the collection name and wouldn't have to change anything else.